frontend/packages/network-attachment-definition-plugin/src/utils/index.ts

```diff
@@ -0,0 +1,21 @@
+import * as _ from 'lodash';
+import {
+  NetworkAttachmentDefinitionConfig,
+  NetworkAttachmentDefinitionKind,
+} from '../components/network-attachment-definitions/types';
+
+export const getConfigAsJSON = (
```

Comment:
this file should be probably called `src/selectors/index.ts`

Reply:
Good call. Done!